### PR TITLE
add A0shrafpoless bio

### PR DESCRIPTION
**Pull request by Ashrafpoless**
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [x] feedback is addressed (if any and if it is appropriate feedback.)

### Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [x] all links and images work
- [x] any embedded code snippets are formatted

### CODE REVIEW

File: student-bios/Ashrafpoless.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# HYF
+
+---
+
+## Ashraf Poless
+
+---
+
+## web developer
```
Comment (denisrossi) [inline student-bios/Ashrafpoless.md:9]: Please, always starts the headers with upper case letter. For instance, here should be `Web Developer` and, at line 19, should be `About me`.

File: student-bios/Ashrafpoless.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+### <ashrafpoless@gmail.com>
+
+---
+
+### 0465 65 56 69
```
Comment (denisrossi) [inline student-bios/Ashrafpoless.md:15]: Not sure what are the policies here @bermarte, but I'd suggest to not make public such personal information like the phone number.

**Comment by bermarte** [comment]
@Ashrafpoless Leave always 2-3 reviewers, please do not unassign them.

File: student-bios/Ashrafpoless.md (hunk 2 above)
Comment (bermarte) [reply student-bios/Ashrafpoless.md:15]: Ok to me, he can delete it.

File: student-bios/Ashrafpoless.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+---
+
+#### about me
+
+## I am a **Hyf** student whose participate an _full stack_ course
```
Comment (bermarte) [inline student-bios/Ashrafpoless.md:21]: I suggest you install Grammarly on your browser, it's really handy.

**Pull request merged**